docs/publishing.md

```diff
@@ -6,53 +6,49 @@ Lerna is a tool that optimizes the workflow around managing multi-package reposi
 
 It manages dependencies between our components so that if the version number of `globals` is bumped, all components dependent on `globals` will also have their version bumped.
 
-### Publishing components manually (while in Alpha)
+### Publishing components manually (while in Alpha/Private beta)
 
 In a new branch:
```

Comment:
Do we want something about making sure you're up to date with the latest changes to master?

Reply:
Added.